Dockerfile

```diff
@@ -1,19 +1,17 @@
 # Use the Anaconda Docker image
 FROM continuumio/anaconda3:5.3.0
 
+# Install build tools, gcc etc
+RUN apt-get update && apt-get install -y build-essential vim htop procps x11vnc xvfb libsdl-ttf2.0-0
```

Comment:
Note: I also re-ordered the Dockerfile slightly to allow for faster cached builds when python dependencies change.